### PR TITLE
New shadow module

### PR DESCRIPTION
**Pull request by niravasher**
Fixes #[Add issue number here.]

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `npm test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] ask `@publiclab/is-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

Thanks!

### CODE REVIEW

**Comment by niravasher** [comment]
@publiclab/is-reviewers please help.

**Comment by HarshKhandeparkar** [comment]
@publiclab/is-reviewers

File: src/modules/Shadow/Module.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        const getPixels = require("get-pixels");
+
+        getPixels(input.src, function(err, pixels) {
+            console.log("nirav");
+            if (options.preProcess) pixels = options.preProcess(pixels); // Allow for preprocessing
+            var pixelsArray = [];
+            var i = 0;
+            var pixel = [];
+            for (var x = 0; x < pixels.shape[0]; x++) {
+                for (var y = 0; y < pixels.shape[1]; y++) {
```
Comment (HarshKhandeparkar) [inline src/modules/Shadow/Module.js]: You don't have to do all of this here. You can do it in extraManipulation without the use of get-pixels. Just put extraManipulation inside draw.

**Comment by jywarren** [comment]
See #994 for the new canvas resize module!!!

**Pull request closed without merging**